packages/frontend/src/components/ChartConfigPanel/FieldLayoutOptions.tsx

```diff
@@ -86,6 +84,21 @@ const FieldLayoutOptions: FC<Props> = ({ items }) => {
         return items.filter((item) => isDimension(item));
     }, [items]);
 
+    const chartInvolvesMetrics = useMemo(() => {
+        if (!validCartesianConfig) return false;
+
+        const {
+            layout: { xField, yField },
+        } = validCartesianConfig;
+
+        if (!xField || !yField) return false;
+
+        const chartAxes = [xField, ...yField];
```

Comment:
> If you have only selected dimensions in your x & y axes, then groups gets greyed out with a helpful message

@TuringLovesDeathMetal The x axis shouldn't matter right? Even if I have a metric in the x axis it doesn't have an impact on the grouping. ( ignoring the case where the user flips axis since we store the data in the `yField` property either way )
So we could just check if the `yField` contains a field that isn't a dimension.

Reply:
yep, axis shouldn't matter!

It's "if you've only selected dimensions as your series" (basically, if you have no metrics in your chart)

Reply:
> So we could just check if the yField contains a field that isn't a dimension.

But that doesn't the cover the case of x-axis with metric
Say, we have x-axis as metric, y-axis as dimension, the grouping should be enabled correct ? ( since there is metric in chart ? )

Reply:
yes? I think so 🤔

Reply:
:ok_hand: Commit f5eb8237a9cb6b7d22a59a81bf151123903ce6f9 should be okay then @ZeRego

Reply:
> Say, we have x-axis as metric, y-axis as dimension, the grouping should be enabled correct ? ( since there is metric in chart ? )

In that case the grouping should be disabled since the X axis has no impact on grouping. 
Explanation of how grouping impacts the chart series:
- We keep 1 series per each dimension in the y axis
- We multiply each y axis field, that is not dimension, per unique values of the grouping dimension.
- 
Eg: 
y axis = ['dimension_a', 'metric_a', 'metric_b']
group field = 'dimension_b'  ( In the results we have the unique values 'x' and 'y' for the field 'dimension_b' )

Series will be:
- dimension_a ( since it is a dimension we don't group/pivot ) 
- metric_a_x ( metric a + value x )
- metric_a_y ( metric a + value y )
- metric_b_x ( metric b + value x )
- metric_b_y ( metric b + value y )

The x axis doesn't have an impact on any of this logic. That's why I'm saying it can be a dimension or a metric or something else.